### ISSUE TITLE
Ui improvement for login and sign up page

### ISSUE THREAD

**Issue by muneebGH** [fossasia/susi_android#2435]
**Describe the chore**
the colour of button in login and sign up page looks disabled . It creates a confusion for newbie .

color should turn to blue like color (same on sushi logo ) if all the input credentials met correctly.
**Would you like to work on the issue?**
I would love to .

**Comment by blackviking991** [comment]
I will like to work please guide me.

**Comment by ankushmundhra** [comment]
@blackviking991 are u working on this?

**Comment by blackviking991** [comment]
Yes I am trying.

**Comment by blackviking991** [comment]
Can you please tell me where do I find these settings in the code base??

**Comment by gagankumre** [comment]
I want to take up this task. I have seen that and I will change the UI a bit like 

- The password prompt should be given based upon user input

- Fonts styles should be changed

- LogIn button colour should change when the input format is correct.

**Comment by iaminamkhan** [comment]
Is this issue resolved?

**Comment by codewithbeny** [comment]
I like to work. please guide me.

**Comment by prayutsu** [comment]
I would like to work on this issue. Can I add material design theme in this project?

**Comment by iamareebjamal** [comment]
There already is

**Comment by prayutsu** [comment]
I would like to add animations and beautify the UI

**Comment by iamareebjamal** [comment]
Not a priority right now

**Comment by ritmojs** [comment]
I would like to work on this issue.

**Comment by MasterBlaster99** [comment]
Yes!!!! i will surely make it beautiful as i have made few apps with really good UI I have gone through the UI and can make it really good

**Comment by thelosergeek** [comment]
Hey @MasterBlaster99 , are you working on the issue? If not, I would like to pick this up!

**Comment by Prashant830** [comment]
can i solve this issue??

**Comment by Aditya8840** [comment]
can i solve this issue??

**Comment by Priti-Palkar** [comment]
Hi 👋, I would like to work on this issue.  
Could you please assign it to me?
